Geometry/VeryForwardGeometryBuilder/plugins/TotemRPGeometryESModule.cc

```diff
@@ -418,7 +418,7 @@ std::unique_ptr<DDCompactView> TotemRPGeometryESModule::produceMeasuredDDCV(cons
 {
     // get the ideal DDCompactView from EventSetup
     edm::ESHandle<DDCompactView> idealCV;
-    iRecord.getRecord<IdealGeometryRecord>().get("XMLIdealGeometryESSource_CTPPS", idealCV);
+    iRecord.getRecord<IdealGeometryRecord>().get(idealCV);
```

Comment:
It is also necessary, when we use MeasuredGeometryRecord instead of IdealGeometryRecord in SimG4. I suppose it's caused by the fact, that we change XMLIdealGeometryESSource_CTPPS for XMLIdealGeometryESSource in our configuration (to solve other problems). @ziaber could you test reconstruction?